src/flags.ts

```diff
@@ -15,6 +18,7 @@ export type IFlagBase<T, I> = {
   required?: boolean;
   dependsOn?: string[];
   exclusive?: string[];
+  exactlyOne?: string[];
```

Comment:
The issue with that is that what happens if you put `onlyOne` and `required` on two or more fields.  

I was trying to avoid mucking around with the logic of `required`, and making this somewhat separate kinda separate.  Given how `required` is currently defined and used, I wasn't sure we'd want to change that paradigm (it's so simple and clean as is)

Reply:
Ya, that's a good point and something I can get behind.

Reply:
the other thing to consider is that "required" shows up in the --help text that may go beyond the scope of /parser.

Reply:
https://github.com/oclif/parser/blob/master/src/help.ts